releases/v1.1.0.toml

```diff
@@ -0,0 +1,150 @@
+listens on the same socket as the containerd GRPC interface and runs in the same
+process.
+
+In addition to all of the stability and bug fixes backported to 1.0,
+1.1 includes...
+
+- CRI plugin
+- ZFS and AUFS snapshotter
+- Improvements to the `ctr` tool
+- Better support for multiple platforms
+- Cross namespace content sharing
+- Better mount cleanup
+- Support for disabling plugins
+- TCP debug address for remote debugging
+- Update to Go 1.10
+- Improvements to the garbage collector
+
+## CRI Plugin
+
+This release of `cri` is a native plugin of `containerd`. It is built into
+`containerd` v1.1 and CRI services are enabled by default.
+
+You can now use Kubernetes, with `containerd` directly, without having to use
+the intermediate `cri-containerd` daemon. The `cri-containerd` daemon is
+end-of-life.
+
+*Note: Please [drain your node](https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/) before
+upgrading from older versions of `cri-containerd` to `containerd` v1.1.*
+
+You can [use a containerd config file to configure the `cri` plugin](https://github.com/containerd/cri/blob/v1.0/docs/config.md).
+
+### Untrusted Workload Runtime
+
+To run an untrusted pod on a runtime for untrusted workload, such as
+[Kata Containers](https://katacontainers.io/) or
+[Clear Containers](https://clearlinux.org/containers), you can:
+1. Configure a runtime for untrusted workload [with the config option `plugins.cri.containerd.untrusted_workload_runtime`](https://github.com/containerd/cri/blob/v1.0/docs/config.md).
+2. Create an untrusted pod by setting the annotation `io.kubernetes.cri.untrusted-workload` to `"true"`, for example:
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx
+  annotations:
+    io.kubernetes.cri.untrusted-workload: "true"
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+```
+
+By default, `cri` will run pods with the default runtime. However, if a pod has
+the `io.kubernetes.cri.untrusted-workload` annotation, the `cri` plugin will run
+the pod with the runtime for untrusted workloads.
+
+Unless configured otherwise, the default runtime is set to
+[runc](https://github.com/opencontainers/runc).
+
+### Container Runtime Interface v1alpha2
+
+The supported CRI (Container Runtime Interface) version for Kubernetes v1.10 is
+now `v1alpha2.` This release of `cri` has been updated to use CRI `v1alpha2`, so
+**it only works with Kubernetes v1.10+.**
+
+New CRI features added in `v1alpha2` are all supported:
+* Container log rotation: Kubelet rotates container logs.
+* Shared pid namespace: Support sharing pid namespace inside a pod.
+
+### Registry Mirror
+
+You can now setup registry configurations with the config option
+`plugins.cri.registry`.
+
+Currently only the `mirrors` option is supported. With it, you can specify
+registry mirrors and secure/insecure connections.
+([doc](https://github.com/containerd/cri/blob/v1.0/docs/registry.md))
```

Comment:
I'll create a branch in `cri`, and this link should point to that branch. Let's remember to update this.

Reply:
Should we point to the tag here? Also is the release branch going to be named `release/1.0`?

Reply:
I've cut the release. Let's update all `v1.0` to `v1.0.0`.